app/services/return-requirements/submit-check-your-answers.service.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+/**
+ * Orchestrates validating the data for `/return-requirements/{sessionId}/check-your-answers` page
+ *
+ * It first retrieves the session instance for the returns requirements journey in progress. It then fetches licence data.
+ * Once licence data has been retrieved we use CheckLicenceEndedService to check if it's either lapsed, expired or revoked.
+ * We throw an error which will be logged caught by Hapi. The user will see the regular error page.
+ *
+ * @param {string} sessionId - The id of the current session
+ *
+ * @returns {string} The licence id
+ */
+const CheckLicenceEndedService = require('./check-licence-ended.service.js')
+const SessionModel = require('../../models/session.model.js')
+const ExpandedError = require('../../errors/expanded.error.js')
+
+async function go (sessionId) {
```

Comment:
Just some tidying up at the top of the module

```suggestion
'use strict'
/**
 * Manages converting the session data to return requirement records when check your answers is confirmed
 * @module SubmitCheckYoursAnswersService
 */

const CheckLicenceEndedService = require('./check-licence-ended.service.js')
const ExpandedError = require('../../errors/expanded.error.js')
const SessionModel = require('../../models/session.model.js')

/**
 * Manages converting the session data to return requirement records when check your answers is confirmed
 * 
 * > This service is work in progress. Some of the functionality described is yet to be implemented
 * 
 * After fetching the session instance for the returns requirements journey in progress it validates that what the user
 * has setup can be persisted for the licence.
 * 
 * If valid it converts the session data to return requirements records then deletes the session record.
 *
 * @param {string} sessionId - The UUID for return requirement setup session record
 *
 * @returns {string} The licence ID
 */
async function go (sessionId) {
```